core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -138,6 +138,10 @@ oppia.factory('SimpleEditorManagerService', [
         data.questionList.addQuestion(QuestionObjectFactory.create(
           lastStateName, stateData.interaction, ''));
       },
+      changeQuestion: function(type,index) {
+        // This change the type of question
```

Comment:
@seanlip I though this will be implemented by person who's first addtional adding question type. Also it'll somewhat tricky, so maybe it's better to push UI changes first.